portfolio.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+ <head>
+	 <link rel="stylesheet" type="text/css" href="normalize.css">
+	 <link rel="stylesheet" type="text/css" href="portfolio.css"  >
+	 <title> Portfolio Simona Ambrosini</title>
+ </head>
+ 
+	
+ <body><
+	 <nav>
+		 <button>Menu</button>
+		 <div class="links">
+	       <a href="navbar.asp" style="text-decoration:none">Home</a>
+	 	   <a href="about.asp" style="text-decoration:none">About</a>
+	 	   <a href="portfolio.asp" style="text-decoration:none">Portfolio</a>
+	 	   <a href="contact.asp" style="text-decoration:none">Contact me</a>
+	 	</div>
+		<div class="download">
+			<a href="|">Download CV</a>
+		</div>
+	 </nav>
+	 <header>
+		 <div class="intro">
+		 <h1>Simona Ambrosini</h1>
+		 <h2></h2>
+	 </div>
+	 <section id="about" class="alt-bg">
+	     <div class="container">
+	         <div class="text">
+	             <h2>About</h2>
+				 <p></p>
+			 </div>
+		 </div>
+	 </section>
+	 <section id="portfolio">
+	     <div class="container"
+	             <h2>Portfolio</h2>
+				 <p></p>
+			 </div>
+		 </div>
+	 </section>
+	 </header>
+	 <img href= >
```

Comment:
I assume this is for testing! ^_^ Also a good rule is to put images and other media content in a `<figure>` element, this allows the `<figcaption>` to remain with the image and makes it super easy for SEO and screen readers to understand